sdk/communication/communication-phone-numbers/test/public/utils/testPhoneNumber.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { env, isPlaybackMode } from "@azure-tools/test-recorder";
+
+const DEFAULT_PHONE_NUMBER = "+14155550100";
+const testAgent = () => env.AZURE_TEST_AGENT;
+const testAgentPhoneNumber = () => env[`AZURE_PHONE_NUMBER_${env.AZURE_TEST_AGENT}`];
```

Comment:
This will return `undefined` if the key is not present in the dictionary. What's the expected behavior in such cases?

Reply:
If this is widely-known through the JS community, I don't think throwing it explicitly would be necessary. Up to you :)